core/zoe/isOfferSafe.js

```diff
@@ -0,0 +1,86 @@
+import { allTrue } from './utils';
+import { insist } from '../../util/insist';
+
+/**
+ * @param  {array} assaysPerIssuer - an array of assays ordered in the
+ * same order as the corresponding issuers
+ * @param  {array} rulesPerIssuer - an array of "rules" ordered in the
+ * same order as the corresponding issuers. Rules are a player's
+ * understanding of the contract that they are entering when they make
+ * an offer. Rules are structured in the form { rule: '', amount}
```

Comment:
put something in the quotes, or replace it with a descriptive name like ruleDescriptionString.